.github/workflows/build_and_test.yml

```diff
@@ -291,12 +291,6 @@ jobs:
           cd docker
           docker compose -f <(envsubst < docker-compose.yml) -f <(envsubst < ${{ matrix.demo_name }}.yml) -p ${PROJECT} stop
 
-      - name: Login to Docker Hub
-        uses: docker/login-action@v2
-        with:
-          username: ${{ secrets.DOCKER_HUB_USERNAME }}
-          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-
```

Comment:
Removing this is necessary, because otherwise forks can not have the CI run on them and this was just a testing as well.

Reply:
Makes sense 👍